ruby/config/Make.rules

```diff
@@ -14,6 +14,7 @@ RUBY ?= ruby
 ruby-call = $(shell $(RUBY) -e 'require "rbconfig"; puts RbConfig::expand("$1")')
 
 # Ruby compiler flags
+platform_cxx    := $(call ruby-call, $$(CXX))
```

Comment:
Where is `platform_cxx` used?

Reply:
I guess that is to use the same compiler as the ruby interpreter?

Reply:
It's used in this file. This is to override the compiler as reconfigure provides the compiler and some flags. 

https://github.com/zeroc-ice/ice/blob/3.7/config/Make.project.rules/#L1146

```
# Ruby 2.6
❯ /usr/bin/ruby -e 'require "rbconfig"; puts RbConfig::expand("$(CXX)")'
xcrun clang++

# Ruby 3.0
~/src/github.com/zeroc-ice/ice/3.7 ruby-3.0
❯ /usr/local/opt/ruby/bin/ruby -e 'require "rbconfig"; puts RbConfig::expand("$(CXX)")'
clang++ -fdeclspec